.github/workflows/fastapi.yml

```diff
@@ -0,0 +1,42 @@
+name: fastapi-testsuite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  fastapi-test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: fastapi
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+      fail-fast: false
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - uses: actions/checkout@v2
+        with:
+          path: ddtrace
+      - uses: actions/checkout@v2
+        with:
+          repository: tiangolo/fastapi
+          ref: 0.63.0
+          path: fastapi
+      - name: Install Flit
+        run: pip install flit
+      - name: Install Dependencies
+        run: flit install --symlink
+      - name: update mypy
+        run: pip install --upgrade --force-reinstall mypy
+      - name: Install Typing
+        run: pip install typing
+      - name: Inject ddtrace
+        run: pip install ddtrace
+      - name: Test
+        run: ddtrace-run scripts/test.sh
```

Comment:
we might want to use something other than `test.sh` here, not sure we want to run their linting?

Reply:
e.g. instead of calling `scripts/test.sh` we just manually call these two lines:

https://github.com/tiangolo/fastapi/blob/22528373bba6a654323de416ad5c867cbadb81bb/scripts/test.sh#L9-L10

we can even modify to remove code coverage reporting, so it will really just be `pytest tests`

and if we control that command (rather than calling through `scripts/test.sh` then we can add our ddtrace plugin to pytest to enable patching/tracing.